ecosystem-modules/ren/renvm-bridge/src/lib.rs

```diff
@@ -113,7 +127,16 @@ decl_module! {
 			sig: EcdsaSignature,
 		) {
 			ensure_none(origin)?;
-			Self::do_mint(who, amount, sig)?;
+			Self::do_mint(&who, amount, &sig)?;
+
+			// Calculated the transaction length from the unit test.
+			let len: u32 = 166;
```

Comment:
make this a `const MINT_TX_LENGTH: u32 = 166` and have test check the const

Reply:
I think the dynamic calculation is better because the amount is compact.